### DISCUSSION TITLE
Thank you ⭐

### DISCUSSION THREAD

**Discussion by johan-overbye** [seerge/g-helper#974] [General]
Thank you @seerge for a perfect little tool. My 2023 Zephyrus is an awesome machine in terms of build quality and performance, but the ASUS software was just completely bewildering and frustrating to use. Now it's easy to love my laptop again, with a clean Windows 11 and G-Helper. Donated. 

Two small issues I noticed:

- Sometimes when i press M4 to open G-Helper, its window appears behind the current window, and I have to alt-tab to it.
- It seems like the keyboard light animation setting is not respected during sleep. Even though it's set to Static, I get a strobing effect of some kind when the laptop is sleeping. I later unticked the "Sleep" box under Laptop Backlight, and that was respected, so now it goes dark during sleep. Zephyrus GU604VI.

Good luck and thanks again!

**Comment by seerge** [comment]
@johan-overbye hello, thanks :)

- Every time main app window is being shown it runs command to set Focus to itself. Rest is up to windows, i can imagine there is some logic there that may prevent annoying windows to pop all the time. Windows sometimes may act weirdly. 
- Sleeping animation is always "strobbing" and always same color on all devices that I know :) it can't changed, only turned ON/ OFF

**Reply by johan-overbye** [reply]
Gotcha, thanks for the info!

**Comment by DomeShifter** [comment]
A big THANK YOU from me too. This software is a dream. Especially compared to the pre-loaded rubbish. I've installed this on my machine and my wife's machine and we love it. Thanks for your hard work and talent @seerge !
